ctsTraffic/ctsConfig.cpp

```diff
@@ -3933,7 +3933,7 @@ namespace ctsTraffic::ctsConfig
 
         if (g_breakOnError)
         {
-            FAIL_FAST_MSG(ctString::convert_to_string(text).c_str());
+            FAIL_FAST_MSG("%hs", ctString::convert_to_string(text).c_str());

```

Comment:
Any reason not to use `"%ws"` and not do the conversion to char based strings in an extra call?

Reply:
Right. Why do that conversion rather than logging the PCWSTR directly with `%ws`?
`FAIL_FAST_MSG` takes a `char*`, but now that you are using a format string, `%ws` can take care of the conversion.

Reply:
Oh, I see what you're saying. Good point.